openapi.yaml

```diff
@@ -16480,14 +16490,18 @@ components:
           type: integer
           minimum: 0
           maximum: 255
-          description: >
-            The priority of the target host for this Record. Lower values are preferred. Only valid and required for SRV record requests.
+          description: |
+            The priority of the target host for this Record. Lower values are preferred. Only valid for
+            MX and SRV record requests. Required for SRV record requests.
+
+            Defaults to `0` for MX record requests. Must be `0` for Null MX records.
```

Comment:
This is correct, and led me to discover a bug. Currently, if the priority isn't provided, then we reject the request before defaulting to `0`. Putting together a fix now; the documentation describes the correct behavior.